changelogs/fragments/4968-pacman-url-integrity-test.yaml

```diff
@@ -0,0 +1,4 @@
+minor_changes:
```

Comment:
Test-only changes shouldn't have a changelog fragment. Changelog fragments are only needed for user-facing changes.